.htaccess

```diff
@@ -1,6 +1,6 @@
 Options -MultiViews
 RewriteEngine On
-RewriteBase /mother_care/
+RewriteBase /
```

Comment:
I rechanged it,
But it wasnt appearing in git status. Will check it again.

On 12 Aug 2016 08:42, "shashank-srikant" <notifications@github.com> wrote:

> In .htaccess
> <https://github.com/shashank-srikant/sevasetu-mothercare/pull/32#discussion_r74536496>
> :
>
> > @@ -1,6 +1,6 @@
> >  Options -MultiViews
> >  RewriteEngine On
> > -RewriteBase /mother_care/
> > +RewriteBase /
>
> @Shubham2801 <https://github.com/Shubham2801>
> can't accept this change.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/shashank-srikant/sevasetu-mothercare/pull/32/files/e449d6eda655cd35afec56c540bf5f3a43c741d2#r74536496>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AT4BkkUpjpaWJsiQxAmRCaq46iZiMspnks5qe-SjgaJpZM4JeUVt>
> .
>